packages/ui-app/src/InputExtrinsic/options/section.ts

```diff
@@ -8,6 +8,21 @@ import { DropdownOptions } from '../types';
 
 import map from '@polkadot/extrinsics';
 
+export function anyMethodToDisplay (methods: any) {
```

Comment:
Since these are re-usable (e.g. same logic applies to eg. storage), put them in seperate files